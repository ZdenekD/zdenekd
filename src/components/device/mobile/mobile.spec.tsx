import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '@/__test__/utils/helpers';
import Mobile from '.';

describe('Mobile device', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Mobile />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-device-mobile');

        expect(component.exists()).toBe(true);
    });
});
