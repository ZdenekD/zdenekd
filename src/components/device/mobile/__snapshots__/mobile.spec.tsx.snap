// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mobile device match snapshot 1`] = `
<div class="wrapper"
     data-test="component-device-mobile"
>
  <div class="mobile">
    <div class="content">
    </div>
    <span class="header">
      <i class="speaker">
      </i>
      <i class="camera">
      </i>
    </span>
    <i class="button"
       style="transform:translateX(50%) translateY(0px) translateZ(0)"
    >
    </i>
    <i class="controlMute">
    </i>
    <i class="controlSound">
    </i>
    <i class="controlSound">
    </i>
    <i class="controlTrigger">
    </i>
  </div>
</div>
`;
